src/main/java/core/basesyntax/model/figure/Circle.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+import javax.persistence.PrimaryKeyJoinColumn;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "circles")
+@PrimaryKeyJoinColumn(name = "circle_id")
```

Comment:
can we go without this annotation in all the entity classes?

Reply:
Yes, if inheritance type isn't defined as JOINED

Reply:
if it is JOINED, this annotation is optional as well

Reply:
Ok, thanks